frontend/packages/helm-plugin/src/catalog/utils/catalog-utils.tsx

```diff
@@ -178,3 +180,12 @@ export const normalizeHelmCharts = (
     [] as CatalogItem[],
   );
 };
+
+export const helmCatalogTypeDescription = ( // add the new QS name in the link
+  <Trans ns="helm-plugin">
+    Browse for charts that help manage complex installations and upgrades. Cluster administrators
+    can customize the content made available in the catalog. Alternatively, developers can try{' '}
+    <Link to={'/quickstart?quickstart='}>this quick start</Link> to configure their own custom Helm
+    Chart repository.
+  </Trans>
+);
```

Comment:
I have some suggestions and a question for this function:

a) I would suggest converting this to a component. Regardless that the component doesn't have a prop, it "feels" more like a small component.

I tested this also with `pseudolocalization` and it looks like it was not translated. I don't know if this static field would work if the user switches the language. I think all these cases should just work when converting to a component.

b) After that would suggest moving this out of the `utils` package into the `components` package. As said, it feels more like a component than a utils function as you need this render context in your test.

Maybe into its own super small file. Would also suggest exporting it then via `helmCatalog` in `packages/helm-plugin/src/catalog/index.ts`. Wdyt?

c) Also mentioned in slack: What happens when the quick start was not found or is disabled? Should we check for the quick start first? :exploding_head: Is it over-engineered if we do this here?

Reply:
moved it out of catalog-utils and created a functional component under components folder